app/server/tests/api-getCommunity.test.js

```diff
@@ -4,31 +4,13 @@ let { AuthFunctions } = require('./functions/authFunctions.js');
 let token = '';
 let user = new AuthFunctions();
 
-const welcomeCommunityTitle = "Welcome";
-const welcomeCommunityDescript = "Test";
-
-
-// Testing the get communities function without logging in
-describe('Get Communities - Without Login', () => {
-  let response = '';
-
-  beforeAll( async() => {
-    response = await community.getCommunities('');
-  });
-
-  test('API returns a unsuccessful response - code 401', () => {
-    expect(response.status).toBe(401);
-  });
-
-  test('API returns description - "Missing token."', () => {
-    expect(response.text).toBe("Missing token.");
-  });
-});
+const welComTitle = "Welcome";
+const welComDescript = "The Neighbourhood";
```

Comment:
Isn't it "The Neighborhood". I believe that's the spelling of the title of the database in mongo.